src/plugins/safeSnap/components/HandleOutcomeUma.vue

```diff
@@ -101,29 +115,29 @@ const approveBondUma = async () => {
     const step = await approveBond.next();
     if (step.value)
       updatePendingTransaction(txPendingId, { hash: step.value.hash });
-    actionInProgress.value = null;
+    action1State.value = 'idle';
     await approveBond.next();
     notify(t('notify.youDidIt'));
     await sleep(3e3);
     await updateDetails();
   } catch (e) {
     console.error(e);
-    actionInProgress.value = null;
+    action1State.value = 'idle';
   } finally {
     removePendingTransaction(txPendingId);
   }
 };
 
-const getProposalUrl = (chain, txHash, logIndex) => {
+const getProposalUrl = (chain: string, txHash: string, logIndex: number) => {
   if (Number(chain) !== 5 && Number(chain) !== 80001) {
-    return `https://oracle.uma.xyz?transactionHash=${txHash}&logIndex=${logIndex}`;
+    return `https://oracle.uma.xyz?transactionHash=${txHash}&eventIndex=${logIndex}`;
```

Comment:
the oracle UI expects the url string to have the param `eventIndex` not `logIndex`